c2corg_ui/static/js/utils.js

```diff
@@ -26,14 +26,17 @@ app.utils.getDoctype = function(type) {
 
 
 /**
- * @param {string} document_type The document type.
+ * @param {string} documentType The document type.
  * @param {string|number} documentId The document id.
  * @param {string} lang Lang.
  * @return {string} Url.
  */
-app.utils.buildDocumentUrl = function(document_type, documentId, lang) {
-  return '/{document_type}/{id}/{lang}'
-    .replace('{document_type}', document_type)
+app.utils.buildDocumentUrl = function(documentType, documentId, lang) {
+  if (!documentType || !documentId || !lang) {
+    return '';
```

Comment:
> To do in another PR?

Sure. Returning `null` also has the advantage that you are correctly handling the case that no url is returned wherever this function is called.